### DISCUSSION TITLE
How can I use atoti as a custom app and use custom logo without Atoti+?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#519] [Q&A] [answered]
Hello guys, I'm planning to use atoti for a long term project I like what atoti offers, it's simple and efficient but I wanna use it permanently with the reports saved without activating Jupyterlab's cell everytime, and I wanna use it as a desktop application any help?

**Comment by HuifangYeo** [comment, marked as answer]
Hi @GameTn,

thanks for using atoti.

You can create a Python project instead of running it in JupyterLab if you do not need visualizations within the notebook.  
Check out the project template: [https://github.com/atoti/project-template](https://github.com/atoti/project-template).

On the other note, maybe we could catch up on how to customize the pivot table as you have asked in a separate channel.

**Reply by GameTn** [reply]
is there any tutorial video i can follow the steps ?

**Reply by HuifangYeo** [reply]
I'm afraid there's no video tutorial at the moment. For the project template, refer to the [README.md](https://github.com/atoti/project-template#readme).

**Reply by HuifangYeo** [reply]
Hi @GameTn, have a look at the atoti documentation, under [deployment](https://docs.atoti.io/latest/deployment/deployment_setup.html).
